src/sentry/models/actor.py

```diff
@@ -179,17 +207,16 @@ def resolve_many(cls, actors: Sequence["ActorTuple"]) -> Sequence[Union["Team",
         return list(filter(None, [results.get((actor.type, actor.id)) for actor in actors]))
 
 
-def handle_actor_pre_save(instance, **kwargs):
+def handle_team_post_save(instance, **kwargs):
     # we want to create an actor if we don't have one
     if not instance.actor_id:
         instance.actor_id = Actor.objects.create(
-            type=ACTOR_TYPES[type(instance).__name__.lower()]
+            type=ACTOR_TYPES[type(instance).__name__.lower()],
```

Comment:
```suggestion
            type=ACTOR_TYPES['team'],
```

I don't think we need to metaprogram this anymore.